entgql/template/mutation_input.tmpl

```diff
@@ -55,6 +55,12 @@ import (
                 {{- end }}
             {{- end }}
         {{- end }}
+
+        {{- with $tmpls := matchTemplate "helper/gql_mutation_input/model/fields/*"  }}
```

Comment:
You have two types of input objects here. Create or Update. 

If already support this, let's add both?

Reply:
You can use `$n.IsCreate` to check if it's `create/fields` or `update/fields`.

Reply:
Yes let's the user developer decide that themselves so it will give them freedom sometimes I could have need the same field to be available in both Create and Update input so I will have to duplicate the code this way if we separate them


User can decided themselves like this
```go
{{ define "helper/gql_mutation_input/model/fields/saasmedia_mutation" }}
	{{- $n := $ }}
		{{- if $n.Annotations.SaasMedia }}
			{{- range $e := $n.Annotations.SaasMedia.Fields }}
				{{- if $n.IsCreate }}

					{{ print ($e.StructField | singular) }}IDs []{{ $e.Type }}
					
					{{- else }}

						{{- $MutationAdd := print "Add" $structField }}
						{{ $MutationAdd }} []string
				{{- end }}
				
			{{- end }}
		{{- end }}
{{ end }}
```